nym-vpn-x/src-tauri/about.toml

```diff
@@ -3,6 +3,7 @@ accepted = [
     "Apache-2.0 WITH LLVM-exception",
     "BSD-2-Clause",
     "BSD-3-Clause",
+    "BSL-1.0",
```

Comment:
Is this https://en.wikipedia.org/wiki/Business_Source_License ? Let me quickly look into it

Reply:
What code is using? As far as I can tell it's not compatible with GPL-3.0?

Reply:
@octol I think it's https://opensource.org/license/bsl-1-0

Reply:
Ah good